### DISCUSSION TITLE
回顾类别的用途是什么?

### DISCUSSION THREAD

**Discussion by xulibin** [Tokinx/Wing#100] [问答]
好像一天之后的笔记都列入回顾里面了，这个分类好像没有太大意义。

不知道是否还有别的用途和作用？

**Comment by ssxzi** [comment]
@Tokinx 求解

**Comment by Tokinx** [comment]
emm，只是一个简单的随机显示N条笔记的功能，如果你的笔记很少就很大概率会随机到最近的笔记。
